cypress/e2e/user.cy.js

```diff
@@ -1,12 +1,39 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import UserPageObject from '../support/pages/user.PageObject';
+
+const userPage = new UserPageObject();
 
 describe('User', () => {
-  before(() => {
+  let user;
+  let secondUser;
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then(generateUser => {
+      user = generateUser;
+    });
+    cy.task('generateUser').then(generateUser => {
+      secondUser = generateUser;
+    });
   });
 
   it.skip('should be able to follow the another user', () => {
```

Comment:
The test case 'should be able to follow another user' is skipped. If the test case is ready, you should remove the 'skip' command to include it in the test run.